include/swift/AST/DiagnosticsSIL.def

```diff
@@ -173,6 +173,9 @@ ERROR(immutable_value_passed_inout,none,
 ERROR(assignment_to_immutable_value,none,
       "immutable value '%0' may not be assigned to",
       (StringRef))
+ERROR(mutating_protocol_witness_method_on_let_constant,none,
+      "cannot perform mutating instruction: '%0' is a 'let' constant",
```

Comment:
It's not an instruction, it's a method call. I think something like `cannot invoke a mutating method on immutable value '%0'` would be more consistent with the other errors.